apps/backend/src/environments/environment.prod.ts

```diff
@@ -3,5 +3,6 @@ export const environment = {
   port: parseInt(process.env.PORT) || 3000,
   host: process.env.HOST || '0.0.0.0',
   key: process.env.JWT_KEY || null,
-  mongo_host: process.env.MONGO_HOST || 'mongodb://localhost:27017'
+  mongo_host: process.env.MONGO_HOST || 'mongodb://localhost:27017',
```

Comment:
esto se resuelve así 
```suggestion
  mongo_host: process.env.MONGO_HOST || 'mongodb://localhost:27017/sala-de-situacion',
```

Reply:
no queda medio choto que quede hardcodeado? De la forma que lo planteo podes enviarlo en una env. , no? por ahí queres que la bd se llame de otra forma :-)

Reply:
claro obvio. eso es un ejemplo
Pero para que querés dos variables de entorno, Si con una esta todo echo. 
Con un solo conexion string podes agregar más parametros opciones. Así planteado quedan obsoletos.
Si armas un archivo .env en el root del projecto te carga las variables de entorno.